File: examples/csharp/contacts/delete_contact.cs
using System;
using System.Diagnostics;
using System.Net.Http;
using System.Threading.Tasks;

namespace MaiAgentExamples.Contacts
{
    public static class DeleteContact
    {
        public static string API_KEY = "<your-api-key>";
        public static string BASE_URL = "https://api.maiagent.ai/api/v1/";

        public static string CONTACT_ID = "<your-contact-id>";

        public static async Task Main(string[] args)
        {
            Debug.Assert(API_KEY != "<your-api-key>", "Please set your API key");
            Debug.Assert(CONTACT_ID != "<your-contact-id>", "Please set your contact id");

            using var httpClient = new HttpClient();
            httpClient.DefaultRequestHeaders.Add("Authorization", $"Api-Key {API_KEY}");

            // 刪除聯絡人（軟刪除）
            var response = await httpClient.DeleteAsync($"{BASE_URL}contacts/{CONTACT_ID}/");
            response.EnsureSuccessStatusCode();

            Console.WriteLine($"聯絡人 {CONTACT_ID} 已刪除");
        }
    }
}

File: examples/python/contacts/delete_contact.py
import requests

API_KEY = '<your-api-key>'
BASE_URL = 'https://api.maiagent.ai/api/v1/'

CONTACT_ID = '<your-contact-id>'

assert API_KEY != '<your-api-key>', 'Please set your API key'
assert CONTACT_ID != '<your-contact-id>', 'Please set your contact id'


def main():
    headers = {'Authorization': f'Api-Key {API_KEY}'}

    # 刪除聯絡人（軟刪除）
    response = requests.delete(
        url=f'{BASE_URL}contacts/{CONTACT_ID}/',
        headers=headers,
    )
    response.raise_for_status()

    print(f"聯絡人 {CONTACT_ID} 已刪除")


if __name__ == '__main__':
    main()
